src/js/hooks/list-pages/outbound/useOutboundFilters.jsx

```diff
@@ -39,8 +41,13 @@ const useOutboundFilters = (sourceType) => {
   }, [currentLocale]);
 
   useEffect(() => {
+    dispatch(fetchAvailableApprovers());
```

Comment:
Won't it be triggered twice now when the location changes?
You have `currentLocation.id` as dependency in two `useEffects`.
I think `currentLocale` should not matter for the users to be refetched? So maybe remove it from here and leave it only in the 48-49 line?

Reply:
`currentLocale` dependency was there before and is probably needed for `fetchRequisitionStatusCodes` and `fetchAvailableApprovers` is not dependent on the currentLocale which is why I put it in a separate useEffect.

The reason why I used `currentLocation.id` instead of `currentLocation` was because it would trigger the useEffect more than one time. I did not go into detail about which exact part of the `currentLocation` object causes the useEffect to trigger but `currentLocation.id` should be the only reason that indicates that the location has changed.

Reply:
I did not notice the `dispatch(fetchAvailableApprovers());` above the comments, so you are right, I need to remove this.